rdl/data_sources/MsSqlDataSource.py

```diff
@@ -111,21 +112,11 @@ def __build_select_statement(self, table_config, columns, batch_config, batch_ke
 
         return f"{select_sql} \n {from_sql} \n {where_sql} \n {order_by_sql};"
 
-    # Returns an array of configured_columns containing only columns that this data source supports. Logs invalid ones.
-    def assert_data_source_is_valid(self, table_config, configured_columns):
+    def get_table_info(self, table_config, last_known_sync_version):
         columns_in_database = self.__get_table_columns(table_config)
-
-        for column in configured_columns:
-            self.__assert_column_exists(column['source_name'],
-                                        columns_in_database,
-                                        f"{table_config['schema']}.{table_config['name']}")
-
-    def __assert_column_exists(self, column_name, columns_in_database, table_name):
-        if column_name in columns_in_database:
-            return True
-
-        message = f'Column {column_name} does not exist in source table {table_name}'
-        raise ValueError(message)
```

Comment:
before, we would barf with the first invalid column.. but now we list all the invalid columns when barfing.